### PR TITLE
Correcting resource type in CODEOWNERS file, and adding a metric explorer workbook

### PR DESCRIPTION
**Pull request by JenLT**
## PR Checklist

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
        Discovered and corrected an erroneous resourceType string in the prior CODEOWNERS PR. 
* [x] Validate your changes using one or more of the [testing](../Documentation/Testing.md) methods.

### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes
* [x] ensure your template has a corresponding gallery entry in the gallery folder
* [x] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [x] ensure all steps have meaningful names
* [x] ensure all parameters and grid columns have display names set so they can be localized
* [x] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [x] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

**Comment by JenLT** [comment]
CODEOWNERS change is to correct an invalid name for our resource type. My apologies.

File: CODEOWNERS [hunk 1]
```diff
@@ -185,7 +185,7 @@
 /gallery/apimanagement-service/** @microsoft/azure-api-management
 /gallery/workbook/microsoft.mobilenetwork-mobilenetwork-sites.json @microsoft/azure-private-mec-workbooks
 /gallery/workbook/microsoft.apimanagement-service.json @microsoft/azure-api-management
-/gallery/workbook/microsoft.experimentation-experimentationworkspaces @microsoft/exp-devs
+/gallery/workbook/Microsoft.Experimentation-experimentWorkspaces @microsoft/exp-devs
```
Comment (gardnerjr) [inline CODEOWNERS:188]: the other gallery files+folders are all lower case.  hypothetically our build/packaging just `tolower`'s everything to be consistent downstream but you may also want to do that to be consistent?  (i want to say client side it also does this?)

**Review by gardnerjr — APPROVED** [review]
approved for codeowners but with a comment there about consistency?

**Comment by JenLT** [comment]
image

[Screenshot of Variant comparison workbook]


image

[Screenshot of Metric explorer workbook]

File: CODEOWNERS (hunk 1 above)
Comment (JenLT) [reply CODEOWNERS]: implemented recommendation

**Review by aprilk-ms — APPROVED** [review]

**Pull request merged**